### PR TITLE
fix(deployments): tweak deployment card styling to match design mock

### PR DESCRIPTION
**Pull request by andrewazores**
Design reference: https://redhat.invisionapp.com/share/YSDL0KRKW#/screens/260764295

styling-before

styling-after

This PR just contains some style changes to the deployment cards so that the layout of elements (status icon, mini pod donut, version label, and menu kebab) match the design.

### CODE REVIEW

**Comment by andrewazores** [comment]
styling-after-2

last commit: added <hr />

**Comment by fabric8cd** [comment]
@andrewazores fabric8/fabric8-ui:SNAPSHOT-PR-2305-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-ui.badger.fabric8.io

**Comment by andrewazores** [comment]
styling-after-3

**Comment by fabric8cd** [comment]
@andrewazores fabric8/fabric8-ui:SNAPSHOT-PR-2305-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@andrewazores fabric8/fabric8-ui:SNAPSHOT-PR-2305-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@andrewazores fabric8/fabric8-ui:SNAPSHOT-PR-2305-6 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-ui.badger.fabric8.io

File: src/app/space/create/deployments/apps/deployment-card.component.html [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
+            <li role="separator" class="divider"></li>
+            <li>
+              <a>Delete</a>
+            </li>
+          </ul>
+        </div>
+      </div>
     </div>
     <div [collapse]="collapsed">
+      <hr />
```
Comment (joshuawilson) [inline src/app/space/create/deployments/apps/deployment-card.component.html]: Controlling spacing with a `<hr />` feels wrong. @mindreeper2420 are you OK with this?

Comment (andrewazores) [reply src/app/space/create/deployments/apps/deployment-card.component.html]: It's not meant to be adding spacing, it's for the grey line that appears between the "collapsed" state content and the "expanded" content.

File: src/app/space/create/deployments/apps/deployment-card.component.less [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.mini-card-content {
+  padding-left: .8rem;
```
Comment (mindreeper2420) [inline src/app/space/create/deployments/apps/deployment-card.component.less]: Either `px` or `em` is preferred over `rem`.

File: src/app/space/create/deployments/apps/deployment-card.component.html [hunk 3]
```diff
@@ -1,31 +1,42 @@
...
-          <a>Open Application</a>
-        </li>
-        <li role="separator" class="divider"></li>
-        <li>
-          <a>Delete</a>
-        </li>
-      </ul>
+    <div class="row mini-card-content">
+      <div class="col-xs-2">
+        <span class="pficon fa-lg pficon-ok" tooltip="Everything looks okay" placement="top"></span>
```
Comment (mindreeper2420) [inline src/app/space/create/deployments/apps/deployment-card.component.html]: write the icon classes using the order 1. family 2. icon name 3. size: `pficon pficon-ok fa-lg`

File: src/app/space/create/deployments/apps/deployment-card.component.html (hunk 1 above)
Comment (mindreeper2420) [reply src/app/space/create/deployments/apps/deployment-card.component.html]: This is the correct usage of this tag. You can remove the trailing slash though - that is not needed `<hr>`

Comment (joshuawilson) [reply src/app/space/create/deployments/apps/deployment-card.component.html]: oh, I missed the grey line. Thanks.

**Comment by fabric8cd** [comment]
@andrewazores fabric8/fabric8-ui:SNAPSHOT-PR-2305-7 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@andrewazores fabric8/fabric8-ui:SNAPSHOT-PR-2305-10 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2305-fabric8-ui.badger.fabric8.io

**Pull request merged**